src/index.html

```diff
@@ -3,11 +3,122 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="
+    width=device-width,
+    user-scalable=no,
+    initial-scale=1.0,
+    maximum-scale=1.0,
+    minimum-scale=1.0
+    "
+  >
+
   <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet"
+    href="./style.css"
+  >
 </head>
+
 <body>
-  <h1>Stars</h1>
+  <div
+    class="
+    star--block
+    stars--0
+    "
+  >
+    <div class="star">
+    </div>
```

Comment:
пустой див не нужно переносить
```suggestion
    <div class="star"></div>
```